lib/resty/core/phase.lua

```diff
@@ -0,0 +1,55 @@
+local ffi = require 'ffi'
+local C = ffi.C
+local base = require "resty.core.base"
+local FFI_ERROR = base.FFI_ERROR
+local tostring = tostring
+
+ffi.cdef[[
+int ngx_http_lua_ffi_get_phase(ngx_http_request_t *r, char **err)
+]]
+
+
+local _M = {
+    _VERSION = base.version
```

Comment:
It seems there are multiple styles at play here in this lib (at least 3 that i can find). See my comment above wrt. uri/hash/exit which seem to exhibit an earlier style. Additionaly, some of the `ngx` modules exhibit conflicting style wrt. cdef indentation. Again, documented style would be very useful for situations like there.